.github/workflows/safety.yml

```diff
@@ -27,4 +27,4 @@ jobs:
         run: python -m poetry install
 
       - name: Run safety dependency check
-        run: python -m poetry run safety check --full-report
+        run: python -m poetry run safety check --full-report -i 70612
```

Comment:
Hum?

Reply:
This is to ignore the jinja2 issue number. We can't fix it by bumping versions since it affects all versions >= 0 ... :/

I can add a comment if you want

Reply:
.. actually it's a pain to add comments in YAML 😭

Reply:
It's okay I just wanted to understand what that does, did not find the option in the docs 😄